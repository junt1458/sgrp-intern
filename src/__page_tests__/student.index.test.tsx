import { render } from '@testing-library/react';
import StudentIndexPage from '../pages/student/index';

it('should render the component', () => {
  const { container } = render(<StudentIndexPage />);
  expect(container).toMatchSnapshot();
});
