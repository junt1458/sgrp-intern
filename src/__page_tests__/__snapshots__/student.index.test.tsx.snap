// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<div>
  <div
    class="flex h-14 w-full items-center justify-between bg-slate-900 px-4 py-2 text-xl"
  >
    Internship Application System
  </div>
  Student Index Page.
</div>
`;
